packages/builders/src/node/build/webpack/config.ts

```diff
@@ -0,0 +1,141 @@
+import * as webpack from 'webpack';
+import { Configuration, ProgressPlugin } from 'webpack';
+
+import * as ts from 'typescript';
+import { dirname, resolve } from 'path';
+
+import { LicenseWebpackPlugin } from 'license-webpack-plugin';
+import CircularDependencyPlugin = require('circular-dependency-plugin');
+import ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
+
+import { BuildNodeBuilderOptions } from '../node-build.builder';
+import * as nodeExternals from 'webpack-node-externals';
+
+export const OUT_FILENAME = 'main.js';
+
+export function getWebpackConfig(options: BuildNodeBuilderOptions) {
+  const webpackConfig: Configuration = {
+    entry: [options.main],
+    mode: options.optimization ? 'production' : 'development',
+    output: {
+      path: options.outputPath,
+      filename: OUT_FILENAME,
+      libraryTarget: 'commonjs'
+    },
+    module: {
+      rules: [
+        {
+          test: /\.ts$/,
+          loader: `ts-loader`,
+          options: {
+            configFile: options.tsConfig,
+            transpileOnly: true,
+            // https://github.com/TypeStrong/ts-loader/pull/685
+            experimentalWatchApi: true
+          }
+        }
+      ]
+    },
+    resolve: {
+      extensions: ['.ts', '.js'],
+      alias: getAliases(options)
+    },
+    target: 'node',
+    node: {
+      console: false,
+      global: false,
+      process: false,
+      Buffer: false,
+      __filename: false,
+      __dirname: false
+    },
+    performance: {
+      hints: false
+    },
+    plugins: [
+      new ForkTsCheckerWebpackPlugin({
+        tsconfig: options.tsConfig,
+        workers: options.maxWorkers || ForkTsCheckerWebpackPlugin.TWO_CPUS_FREE
+      })
+    ],
+    watch: options.watch
+  };
+
+  const extraPlugins: webpack.Plugin[] = [];
+
+  if (options.progress) {
+    extraPlugins.push(new ProgressPlugin());
+  }
+
+  if (options.optimization) {
```

Comment:
As with the above statement about `mode` I don't think there is any need for this.